pkg/transport/http/transport.go

```diff
@@ -445,8 +445,8 @@ func createNonGeneralAuthSignature(wallet wallet.WalletInterface, initialNonce,
 }
 
 func setupContext(req *http.Request, requestData *transport.AuthMessage, requestID string) *http.Request {
-	ctx := context.WithValue(req.Context(), transport.IdentityKey, requestData.IdentityKey) //nolint:staticcheck // we want to use the key as a static string
-	ctx = context.WithValue(ctx, transport.RequestID, requestID)                            //nolint:staticcheck // we want to use the key as a static string
+	ctx := context.WithValue(req.Context(), transport.IdentityKey, requestData.IdentityKey)
+	ctx = context.WithValue(ctx, transport.RequestID, requestID) //nolint:staticcheck // we want to use the key as a static string
```

Comment:
Why here you want use the simply string, while for IdentityKey, you are using custom type?